modules/ftui/ftui.helper.js

```diff
@@ -227,6 +226,66 @@ export function durationFromSeconds(time) {
   return ret;
 }
 
+export function timeFormat(ms, format, imode='ms', fmode='lower') {
+  // imode: 'ms' or 's'
+  let x = ms
+  if (imode == 'ms') { x /= 1000; }
+
+  const SECONDS = ~~(x);
+  const seconds = ~~(x % 60);
+  x /= 60;
+  const MINUTES = ~~(x);
+  const minutes = ~~(x % 60);
+  x /= 60;
+  const HOURS = ~~(x);
```

Comment:
Please only use upper case names for constants like PI = 3.1415; or MINUTES = 60, not for variables.

https://www.w3schools.com/js/js_conventions.asp

Reply:
Now using totalSeconds, totalMinutes, totalHours, totalDays.